### DISCUSSION TITLE
Talk about everything you dislike in the game in here

### DISCUSSION THREAD

**Discussion by royalealbert27** [studio-minus/ppg-bugs-and-requests#2284] [General]
rule 1: no violent argruments
rule 2: respect eachother
rule 3: put the reason why you don't like the thing you don't like

**Comment by Drogo-man** [comment]
I hate it when the humans die and blood comes everywhere, so violent. I wish Zooi would add Gardening.

**Reply by dinevolence** [reply]
Then make a feature request if you wish for it to be added.

**Comment by ghost** [comment]
I get infuriated when I kill a human and his body **INSTANTLY** goes 1% rotted

**Comment by thetruss** [comment]
I hate when a knife doesn't pierce objects and ends up pushing it away instead

**Comment by ATPGaming** [comment]
I hate it when a human falls over, instead of, I don't know, bending his knees, he kicks the ground and hurts himself until he gets to his feet

**Comment by TeslaCoil9001** [comment]
When humans die, this game is so violent!!1 Not for kids... 😡😡😡😡😡😡😡😡😡😡

**Comment by mestiez** [comment]
i hate when i have to open the unity editor to fix anything

**Reply by Drogo-man** [reply]
Same bruh, i tried to get into unity but i kept getting compiler errors so i cried myself to sleep and never touched it again.
My life is way better without unity.

**Comment by unfuny1337** [comment]
i dont like stab system it works weird sometimes

**Comment by inferknob** [comment]
how zooi ignores good feature requests (a cry for help)
